test/dlc_tests/sanity/test_utility_installation.py

```diff
@@ -84,6 +104,28 @@ def test_utility_packages_using_import(training):
             assert Version(version) > Version(
                 "2"
             ), f"Sagemaker version should be > 2.0. Found version {version}"
+            
+
+@pytest.mark.model("N/A")
+@pytest.mark.integration("common pytorch training utility packages")
+def test_commmon_pytorch_utility_packages_using_import(pytorch_training):
+    """
+    Verify that common utility packages are installed in the Training DLC image
+    :param training: training ECR image URI
+    """
+
+    ctx = Context()
+    container_name = test_utils.get_container_name("commmon_pytorch_utility_packages_using_import", pytorch_training)
+    test_utils.start_container(container_name, pytorch_training, ctx)
+    packages_to_import = COMMON_PYTORCH_TRAINING_UTILITY_PACKAGES_IMPORT
+
+    for package in packages_to_import:
+        test_utils.run_cmd_on_container(
+            container_name,
+            ctx,
+            f"import {package}; print({package}.__version__)",
+            executable="python",
+        )
```

Comment:
Thank you for adding this test!

Could you slightly change the structure of this loop so that it runs a `try-except` on each package being imported, and then collects a list of packages whose imports failed?

This is so that a failure to import, say, `PIL` doesn't prevent a failure in importing `boto3` from surfacing until the `PIL` issue is fixed.

Reply:
Please also run `black -l 100` to reformat the file.

Reply:
done